src/to_string.c

```diff
@@ -124,6 +124,25 @@ size_t number_to_string(char *const dest, uint64_t number) {
     return length;
 }
 
+static const char contract_type_present[] = "Present";
+static const char contract_type_not_present[] = "Not Present";
+
+void contract_type_to_string_indirect(char *const dest, size_t const buff_size, uint8_t const *const contract_type) {
+    check_null(dest);
+
+    if (*contract_type != CONTRACT_NOT_PRESENT) {
+        if (sizeof(contract_type_present) >= buff_size)
+            THROW(EXC_WRONG_LENGTH);
+        memcpy(dest, contract_type_present, sizeof(contract_type_present));
+        dest[strlen(contract_type_present)] = '\0';
```

Comment:
Not needed because string literal should already do null byte. https://stackoverflow.com/questions/20161921/null-byte-and-arrays-in-c

Reply:
does sizeof(contract_type_present) include the null byte?